docker-compose.override.yaml

```diff
@@ -0,0 +1,41 @@
+version: '3.4'
+services:
+  mine-support-payment-service:
+    build: .
+    image: mine-support-payment-service
+    container_name: mine-support-payment-service
+    command: sh -c "scripts/wait-for/wait-for mine-support-payment-postgres:5432 -- scripts/wait-for/wait-for mine-support-payment-message-queue:8161 -- npm run migrate && npm run start:watch"
+    depends_on:
+      - mine-support-payment-message-queue
+    volumes:
+      - node_modules:/usr/src/app/node_modules
+      - .:/usr/src/app
+
+  mine-support-payment-message-queue:
+    container_name: mine-support-payment-message-queue
+    image: vromero/activemq-artemis:2.9.0-alpine
+    environment:
+      ARTEMIS_USERNAME: artemis
+      ARTEMIS_PASSWORD: artemis
+    networks:
+      - mine-support-payment
+    ports:
+      - "8161:8161"
+      - "5672:5672"
+    volumes:
+      - message-queue-data:/var/lib/artemis/data
+      - message-queue-tmp:/var/lib/artemis/tmp
+      - message-queue-etc:/var/lib/artemis/etc
+      - message-queue-etc-override:/var/lib/artemis/etc-override
+      - message-queue-lock:/var/lib/artemis/lock

```

Comment:
Each of these named volumes is mounted to a location declared as a volume in the activemq-artemis Dockerfile. These are specified so that the volumes are all named and will be cleaned up by `docker-compose down -v`.